browser-sync/browser-sync.d.ts

```diff
@@ -371,7 +373,7 @@ declare module "browser-sync" {
              * The stream method returns a transform stream and can act once or on many files.
              * @param opts Configuration for the stream method
              */
-            stream(opts?: { once: boolean }): NodeJS.ReadWriteStream;
+            stream(opts?: { once?: boolean, match?: string | RegExp | ((testString: string) => boolean) | (string | RegExp | ((testString: string) => boolean))[] }): NodeJS.ReadWriteStream;
```

Comment:
What about `StreamOptions` instead of `MatchOptions`, as it's not just match options?